src/index.js

```diff
@@ -8,6 +8,7 @@ process.on('unhandledRejection', (reason, p) =>
   logger.error('Unhandled Rejection at: Promise ', p, reason)
 );
 
-server.on('listening', () =>
-  logger.info('Feathers application started on http://%s:%d', app.get('host'), port)
-);
+server.on('listening', () => {
+  logger.info('Feathers application started on http://%s:%d', app.get('host'), port);
+  app.service('amqp-connector').receiveFromQueue(process.env.RABBITMQ_HOST, app.get('rabbitQueue'));
```

Comment:
Why is this an environment variable and not part of the config file?

Reply:
Also, I don't think this call really belongs to the `listening` callback. Couldn't it be placed below. the callback?